unlock-app/src/components/interface/verification/Scanner.tsx

```diff
@@ -169,6 +175,8 @@ export function Scanner() {
                 <div className="flex items-center justify-center min-h-full">
                   <Dialog.Panel className="w-full max-w-sm">
                     <VerificationStatus
+                      eventAddresses={eventAddresses}
```

Comment:
I think it is much better to pass the paywallconfig here so we always have the same kind of object in the app. The addresses themselves may not be enough (we may need the network...)